src/Agent.Sdk/Util/LoggedSecretMasker.cs

```diff
@@ -71,7 +71,8 @@ public void AddRegex(string pattern, string origin)
         }
 
         // We don't allow to skip secrets longer than 4 characters.
```

Comment:
Probably need to change 4 to 6?

Reply:
Actually it should be 5 😄 
Fixed, thanks!